.github/scripts/artifact-upload-params.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+RUNTIME_PATH='target/release/wbuild/humanode-runtime/humanode_runtime.compact.compressed.wasm'
+
+case "$ARTIFACT_SELECTOR" in
```

Comment:
Selector should only be responsible for selecting the artifact path, not for naming the artifact (that can be addressed separately - like by adding another knob to the mode)